public/content/it/welcome/welcome-to-d.md

```diff
@@ -0,0 +1,58 @@
+# Benvenuto al tour di D
+
+Benvenuto al tour interattivo del *Linguaggio di Programmazione D*.
```

Comment:
You have a point. I'll switch to "Benvenuto nel" as it gives a better feeling to the reader. Thanks for the feedback.